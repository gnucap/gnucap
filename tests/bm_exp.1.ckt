' exp test
.option out=170
v1 1 0 exp  iv= 0.  pv= 1.  td1= 1.n  tau1= 5.n  td2= 10.n  tau2= 5.n
.print tran v 1
.tran 0 20n .05n
.list
*>.status notime
.end
