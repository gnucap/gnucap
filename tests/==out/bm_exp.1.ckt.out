' exp test 
#Time       v(1)      
 0.         0.        
 50.p       0.        
 100.p      0.        
 150.p      0.        
 200.p      0.        
 250.p      0.        
 300.p      0.        
 350.p      0.        
 400.p      0.        
 450.p      0.        
 500.p      0.        
 550.p      0.        
 600.p      0.        
 650.p      0.        
 700.p      0.        
 750.p      0.        
 800.p      0.        
 850.p      0.        
 900.p      0.        
 950.p      0.        
 1.n        0.        
 1.05n      0.0099502 
 1.1n       0.019801  
 1.15n      0.029554  
 1.2n       0.039211  
 1.25n      0.048771  
 1.3n       0.058235  
 1.35n      0.067606  
 1.4n       0.076884  
 1.45n      0.086069  
 1.5n       0.095163  
 1.55n      0.10417   
 1.6n       0.11308   
 1.65n      0.1219    
 1.7n       0.13064   
 1.75n      0.13929   
 1.8n       0.14786   
 1.85n      0.15634   
 1.9n       0.16473   
 1.95n      0.17304   
 2.n        0.18127   
 2.05n      0.18942   
 2.1n       0.19748   
 2.15n      0.20547   
 2.2n       0.21337   
 2.25n      0.2212    
 2.3n       0.22895   
 2.35n      0.23662   
 2.4n       0.24422   
 2.45n      0.25174   
 2.5n       0.25918   
 2.55n      0.26655   
 2.6n       0.27385   
 2.65n      0.28108   
 2.7n       0.28823   
 2.75n      0.29531   
 2.8n       0.30232   
 2.85n      0.30927   
 2.9n       0.31614   
 2.95n      0.32294   
 3.n        0.32968   
 3.05n      0.33635   
 3.1n       0.34295   
 3.15n      0.34949   
 3.2n       0.35596   
 3.25n      0.36237   
 3.3n       0.36872   
 3.35n      0.375     
 3.4n       0.38122   
 3.45n      0.38737   
 3.5n       0.39347   
 3.55n      0.3995    
 3.6n       0.40548   
 3.65n      0.4114    
 3.7n       0.41725   
 3.75n      0.42305   
 3.8n       0.42879   
 3.85n      0.43447   
 3.9n       0.4401    
 3.95n      0.44567   
 4.n        0.45119   
 4.05n      0.45665   
 4.1n       0.46206   
 4.15n      0.46741   
 4.2n       0.47271   
 4.25n      0.47795   
 4.3n       0.48315   
 4.35n      0.48829   
 4.4n       0.49338   
 4.45n      0.49842   
 4.5n       0.50341   
 4.55n      0.50836   
 4.6n       0.51325   
 4.65n      0.51809   
 4.7n       0.52289   
 4.75n      0.52763   
 4.8n       0.53233   
 4.85n      0.53699   
 4.9n       0.54159   
 4.95n      0.54616   
 5.n        0.55067   
 5.05n      0.55514   
 5.1n       0.55957   
 5.15n      0.56395   
 5.2n       0.56829   
 5.25n      0.57259   
 5.3n       0.57684   
 5.35n      0.58105   
 5.4n       0.58522   
 5.45n      0.58934   
 5.5n       0.59343   
 5.55n      0.59748   
 5.6n       0.60148   
 5.65n      0.60545   
 5.7n       0.60937   
 5.75n      0.61326   
 5.8n       0.61711   
 5.85n      0.62092   
 5.9n       0.62469   
 5.95n      0.62842   
 6.n        0.63212   
 6.05n      0.63578   
 6.1n       0.63941   
 6.15n      0.64299   
 6.2n       0.64655   
 6.25n      0.65006   
 6.3n       0.65354   
 6.35n      0.65699   
 6.4n       0.6604    
 6.45n      0.66378   
 6.5n       0.66713   
 6.55n      0.67044   
 6.6n       0.67372   
 6.65n      0.67697   
 6.7n       0.68018   
 6.75n      0.68336   
 6.8n       0.68651   
 6.85n      0.68963   
 6.9n       0.69272   
 6.95n      0.69578   
 7.n        0.69881   
 7.05n      0.7018    
 7.1n       0.70477   
 7.15n      0.70771   
 7.2n       0.71062   
 7.25n      0.7135    
 7.3n       0.71635   
 7.35n      0.71917   
 7.4n       0.72196   
 7.45n      0.72473   
 7.5n       0.72747   
 7.55n      0.73018   
 7.6n       0.73286   
 7.65n      0.73552   
 7.7n       0.73815   
 7.75n      0.74076   
 7.8n       0.74334   
 7.85n      0.74589   
 7.9n       0.74842   
 7.95n      0.75092   
 8.n        0.7534    
 8.05n      0.75586   
 8.1n       0.75829   
 8.15n      0.76069   
 8.2n       0.76307   
 8.25n      0.76543   
 8.3n       0.76776   
 8.35n      0.77007   
 8.4n       0.77236   
 8.45n      0.77463   
 8.5n       0.77687   
 8.55n      0.77909   
 8.6n       0.78129   
 8.65n      0.78346   
 8.7n       0.78562   
 8.75n      0.78775   
 8.8n       0.78986   
 8.85n      0.79195   
 8.9n       0.79402   
 8.95n      0.79607   
 9.n        0.7981    
 9.05n      0.80011   
 9.1n       0.8021    
 9.15n      0.80407   
 9.2n       0.80602   
 9.25n      0.80795   
 9.3n       0.80986   
 9.35n      0.81175   
 9.4n       0.81363   
 9.45n      0.81548   
 9.5n       0.81732   
 9.55n      0.81913   
 9.6n       0.82093   
 9.65n      0.82272   
 9.7n       0.82448   
 9.75n      0.82623   
 9.8n       0.82796   
 9.85n      0.82967   
 9.9n       0.83136   
 9.95n      0.83304   
 10.n       0.8347    
 10.05n     0.8264    
 10.1n      0.81817   
 10.15n     0.81003   
 10.2n      0.80197   
 10.25n     0.79399   
 10.3n      0.78609   
 10.35n     0.77827   
 10.4n      0.77053   
 10.45n     0.76286   
 10.5n      0.75527   
 10.55n     0.74775   
 10.6n      0.74031   
 10.65n     0.73295   
 10.7n      0.72565   
 10.75n     0.71843   
 10.8n      0.71129   
 10.85n     0.70421   
 10.9n      0.6972    
 10.95n     0.69026   
 11.n       0.6834    
 11.05n     0.6766    
 11.1n      0.66986   
 11.15n     0.6632    
 11.2n      0.6566    
 11.25n     0.65007   
 11.3n      0.6436    
 11.35n     0.63719   
 11.4n      0.63085   
 11.45n     0.62458   
 11.5n      0.61836   
 11.55n     0.61221   
 11.6n      0.60612   
 11.65n     0.60009   
 11.7n      0.59412   
 11.75n     0.5882    
 11.8n      0.58235   
 11.85n     0.57656   
 11.9n      0.57082   
 11.95n     0.56514   
 12.n       0.55952   
 12.05n     0.55395   
 12.1n      0.54844   
 12.15n     0.54298   
 12.2n      0.53758   
 12.25n     0.53223   
 12.3n      0.52693   
 12.35n     0.52169   
 12.4n      0.5165    
 12.45n     0.51136   
 12.5n      0.50627   
 12.55n     0.50123   
 12.6n      0.49625   
 12.65n     0.49131   
 12.7n      0.48642   
 12.75n     0.48158   
 12.8n      0.47679   
 12.85n     0.47204   
 12.9n      0.46735   
 12.95n     0.4627    
 13.n       0.45809   
 13.05n     0.45354   
 13.1n      0.44902   
 13.15n     0.44455   
 13.2n      0.44013   
 13.25n     0.43575   
 13.3n      0.43142   
 13.35n     0.42712   
 13.4n      0.42287   
 13.45n     0.41867   
 13.5n      0.4145    
 13.55n     0.41038   
 13.6n      0.40629   
 13.65n     0.40225   
 13.7n      0.39825   
 13.75n     0.39428   
 13.8n      0.39036   
 13.85n     0.38648   
 13.9n      0.38263   
 13.95n     0.37882   
 14.n       0.37506   
 14.05n     0.37132   
 14.1n      0.36763   
 14.15n     0.36397   
 14.2n      0.36035   
 14.25n     0.35676   
 14.3n      0.35321   
 14.35n     0.3497    
 14.4n      0.34622   
 14.45n     0.34277   
 14.5n      0.33936   
 14.55n     0.33599   
 14.6n      0.33264   
 14.65n     0.32933   
 14.7n      0.32606   
 14.75n     0.32281   
 14.8n      0.3196    
 14.85n     0.31642   
 14.9n      0.31327   
 14.95n     0.31016   
 15.n       0.30707   
 15.05n     0.30401   
 15.1n      0.30099   
 15.15n     0.29799   
 15.2n      0.29503   
 15.25n     0.29209   
 15.3n      0.28919   
 15.35n     0.28631   
 15.4n      0.28346   
 15.45n     0.28064   
 15.5n      0.27785   
 15.55n     0.27508   
 15.6n      0.27235   
 15.65n     0.26964   
 15.7n      0.26695   
 15.75n     0.2643    
 15.8n      0.26167   
 15.85n     0.25906   
 15.9n      0.25649   
 15.95n     0.25393   
 16.n       0.25141   
 16.05n     0.24891   
 16.1n      0.24643   
 16.15n     0.24398   
 16.2n      0.24155   
 16.25n     0.23915   
 16.3n      0.23677   
 16.35n     0.23441   
 16.4n      0.23208   
 16.45n     0.22977   
 16.5n      0.22748   
 16.55n     0.22522   
 16.6n      0.22298   
 16.65n     0.22076   
 16.7n      0.21856   
 16.75n     0.21639   
 16.8n      0.21424   
 16.85n     0.2121    
 16.9n      0.20999   
 16.95n     0.2079    
 17.n       0.20583   
 17.05n     0.20379   
 17.1n      0.20176   
 17.15n     0.19975   
 17.2n      0.19776   
 17.25n     0.1958    
 17.3n      0.19385   
 17.35n     0.19192   
 17.4n      0.19001   
 17.45n     0.18812   
 17.5n      0.18625   
 17.55n     0.18439   
 17.6n      0.18256   
 17.65n     0.18074   
 17.7n      0.17894   
 17.75n     0.17716   
 17.8n      0.1754    
 17.85n     0.17366   
 17.9n      0.17193   
 17.95n     0.17022   
 18.n       0.16852   
 18.05n     0.16685   
 18.1n      0.16519   
 18.15n     0.16354   
 18.2n      0.16192   
 18.25n     0.1603    
 18.3n      0.15871   
 18.35n     0.15713   
 18.4n      0.15557   
 18.45n     0.15402   
 18.5n      0.15249   
 18.55n     0.15097   
 18.6n      0.14947   
 18.65n     0.14798   
 18.7n      0.14651   
 18.75n     0.14505   
 18.8n      0.14361   
 18.85n     0.14218   
 18.9n      0.14076   
 18.95n     0.13936   
 19.n       0.13798   
 19.05n     0.1366    
 19.1n      0.13524   
 19.15n     0.1339    
 19.2n      0.13257   
 19.25n     0.13125   
 19.3n      0.12994   
 19.35n     0.12865   
 19.4n      0.12737   
 19.45n     0.1261    
 19.5n      0.12485   
 19.55n     0.1236    
 19.6n      0.12237   
 19.65n     0.12116   
 19.7n      0.11995   
 19.75n     0.11876   
 19.8n      0.11757   
 19.85n     0.1164    
 19.9n      0.11525   
 19.95n     0.1141    
 20.n       0.11296   
v1 ( 1 0 ) exp iv= 0. pv= 1. td1= 1.n tau1= 5.n td2= 10.n tau2= 5.n
Gnucap   System status
iterations: op=0, dc=0, tran=1184, fourier=0, total=1586
transient timesteps: accepted=402, rejected=0, total=402
nodes: user=1, subckt=0, model=0, total=1
dctran density=100.0%, ac density=100.0%
